README.md

```diff
@@ -36,3 +36,197 @@ The `storage` subproject contains code related to the key-value storage of the R
+[info]  ---> 596c9fcbc8fc
+[info] Step 4/5 : ADD 1/main.sh /bin
+[info]  ---> ee310eb8b156
+[info] Step 5/5 : ENTRYPOINT ["\/bin\/main.sh"]
+[info]  ---> Running in d2874d6b814a
+[info] Removing intermediate container d2874d6b814a
+[info]  ---> 08e1d3b9f33a
+[info] Successfully built 08e1d3b9f33a
+[info] Tagging image 08e1d3b9f33a with name: coop.rchain/coop.rchain-node:latest
+[info] Tagging image 08e1d3b9f33a with name: coop.rchain/coop.rchain-node:v0.1
+[success] Total time: 17 s, completed Feb 13, 2018 4:39:20 PM
+sbt:rchain>
+```
+
+A single-line build command that does the trick is
+```
+<computer:~/src/rchain (dev)> sbt clean bnfc:generate compile docker
+[... lots of output snipped ...]
+[info] Step 5/5 : ENTRYPOINT ["\/bin\/main.sh"]
+[info]  ---> Running in 015ae98a7ea7
+[info] Removing intermediate container 015ae98a7ea7
+[info]  ---> 33341b27ac61
+[info] Successfully built 33341b27ac61
+[info] Tagging image 33341b27ac61 with name: coop.rchain/coop.rchain-node:latest
+[info] Tagging image 33341b27ac61 with name: coop.rchain/coop.rchain-node:v0.1
+[success] Total time: 9 s, completed Feb 14, 2018 7:47:59 AM
+```
+
+### Running
+
+Invoking the above Docker image is simple enough:
+```
+<computer:~/src/rchain (dev)> docker run -ti --net=host coop.rchain/coop.rchain-node:latest
+15:49:21.363 [main] INFO main - uPnP: Some(/10.0.0.9) -> Some(192.168.0.101)
+15:49:21.497 [main] INFO main - Listening for traffic on #{Network rnode://facff6c005814a669b2063b38f8fc6c4@10.0.0.9:30304}.
+15:49:21.500 [main] INFO main - Bootstrapping from #{PeerNode 0f365f1016a54747b384b386b8e85352}.
+15:49:21.500 [main] DEBUG p2p - connect(): Connecting to #{PeerNode 0f365f1016a54747b384b386b8e85352}
+15:49:21.826 [main] DEBUG p2p - connect(): Received encryption handshake response from #{PeerNode 0f365f1016a54747b384b386b8e85352}.
+15:49:21.868 [main] DEBUG p2p - connect(): Received protocol handshake response from #{PeerNode 0f365f1016a54747b384b386b8e85352}.
+15:49:25.715 [Thread-2] INFO p2p - Responded to encryption handshake request from #{PeerNode bfa3c0b9d9ce4f30adc2e17979f93285}.
+15:49:25.882 [Thread-2] INFO p2p - Responded to protocol handshake request from #{PeerNode bfa3c0b9d9ce4f30adc2e17979f93285}.
+15:49:26.974 [main] INFO main - Possibly new peer: #{PeerNode c12882b563fa47c9af297ce952ef7d94}.
+15:49:26.974 [main] DEBUG p2p - connect(): Connecting to #{PeerNode c12882b563fa47c9af297ce952ef7d94}
+[...]
+```
+Each scoped build is as similar to the original, per-project build process as possible, so assemblies should be produced in the same way as before:
+```
+<computer:~/src/rchain (dev)> sbt "project rholang" assembly
+[info] Loading settings from plugins.sbt ...
+[... compiling, testing, jarring ...]
+[info] Packaging /home/kirkwood/src/rchain/rholang/target/scala-2.12/rholang-assembly-0.1.0-SNAPSHOT.jar ...
+<computer:~/src/rchain (dev)> java -jar rholang/target/scala-2.12/rholang-assembly-0.1.0-SNAPSHOT.jar rholang/examples/hello_world_again.rho 
```

Comment:
We should probably recommend the bash script that @dckc wrote.
```
$ ./rho2rbl examples/hello_world_again.rho 
compiled examples/hello_world_again.rho to examples/hello_world_again.rbl
```
Otherwise we might as well delete it.